common/static/js/xblock/core.js

```diff
@@ -46,36 +46,58 @@
         var block;
         var $element = $(element);
         var runtime = elementRuntime(element);
+        var deferred = $.Deferred();
+        var initialized = $.Deferred();
 
         block_args.unshift(element);
         block_args.unshift(runtime);
 
+        var true_require = require || RequireJS.require;
```

Comment:
I don't understand this variable. require was already mapped to require || RequireJS.requre on line 164. I think this variable can just be deleted.